src/compas_view2/views/view120.py

```diff
@@ -26,6 +28,13 @@ def init(self):
         self.shader.uniform3f("selection_color", self.selection_color)
         self.shader.release()
 
+        self.shader_text = Shader(name='120/text')
+        self.shader_text.bind()
+        self.shader_text.uniform4x4("projection", self.camera.projection(self.app.width, self.app.height))
+        self.shader_text.uniform4x4("viewworld", self.camera.viewworld())
+        self.shader_text.uniform1f("opacity", self.opacity)
+        self.shader_text.release()
+
     def resize(self, w, h):
         self.shader.bind()
```

Comment:
should we rename the default shader to `shader_model` or something like that as well?

Reply:
yep sounds good will do it